eth2deposit/utils/intl.py

```diff
@@ -0,0 +1,85 @@
+import inspect
+import difflib
+from functools import reduce
+import json
+from typing import (
+    Any,
+    Dict,
+    Iterable,
+    List,
+    Mapping,
+    Sequence,
+)
+import os
+
+from eth2deposit.utils import config
+from eth2deposit.utils.constants import (
+    INTL_CONTENT_PATH,
+)
+from eth2deposit.exceptions import ValidationError
+
+
+def _get_from_dict(dataDict: Dict[str, Any], mapList: Iterable[str]) -> str:
+    '''
+    Iterate nested dictionaries
+    '''
+    try:
+        return reduce(dict.get, mapList, dataDict)  # type: ignore
+    except TypeError:
+        raise KeyError('%s not in internationalisation json file.' % mapList)
+
+
+def load_text(params: List[str], file_path: str='', func: str='', lang: str='') -> str:
+    '''
+    Determine and return the appropriate internationalisation text for a given set of `params`.
+    '''
+    if file_path == '':
+        # Auto-detect file-path based on call stack
+        file_path = inspect.stack()[1].filename
+        file_path = file_path[:-3] + '.json'     # replace .py with .json
+
+    if func == '':
+        # Auto-detect function based on call stack
+        func = inspect.stack()[1].function
+
+    if lang == '':
+        lang = config.language
+
+    # Determine path to json text
```

Comment:
The software would break if the target string does not exist in the target language JSON file.
I think in future iterations, we might update the English string during development, and then update the language pack later. What do you think about making it:
1. Try to find the target string in the target language JSON file.
2. If it can't find, try to find the target string in the English JSON file.

Reply:
If we are missing a field accidentally, hopefully the JSON schema tests I implemented will catch that scenario (provided the schemas themselves are up to date).

Reverting to English in the case of a failure is not very intl friendly, however it does work well for ensuring that the CLI remains working. I agree with your argument around making the CLI more resilient to language changes that propagate from english to the rest of the pack.

Reply:
Added English fall-over in https://github.com/ethereum/eth2.0-deposit-cli/pull/182/commits/3a06e92822043e56c37744fa2d34088ab9e58dbe